src/main/java/com/google/cloud/tools/maven/RunMojo.java

```diff
@@ -40,11 +43,18 @@
   /**
    * Path to a yaml file, or a directory containing yaml files, or a directory containing
    * WEB-INF/web.xml.
+   *
+   * @deprecated 1.2.2
```

Comment:
Yeah, although it'd be nice to have a better reference of what is expected here by Maven. According to the [docs](http://maven.apache.org/ref/3.2.1/maven-plugin-api/plugin.html):
>Specify the version when the Mojo was deprecated to the API. Similar to Javadoc deprecated. This will trigger a warning when a user tries to configure a parameter marked as deprecated.

However this doesn't seem to work in my local development, so we could just document the reason along with the version, and display the warning in the `execute()` method as it is now.